tests/framework/clients/notification.go

```diff
@@ -56,3 +58,15 @@ func (t *NotificationOperation) CheckNotificationCR(notificationName string) boo
 
 	return true
 }
+
+func (t *NotificationOperation) CheckNotificationFromHTTPEndPoint(appLabel, eventName, fileName string) (bool, error) {
+	selectorName := "--selector=" + appLabel
```

Comment:
maybe we should call sleep before wh check for the notification. it may take some time (at least with persistent notifications) for the notifications to be sent to the HTTP server, and appear in the log